### ISSUE TITLE
Show an error message when trying to upload files bigger than upload_max_filesize

### ISSUE THREAD

**Issue by laherreria** [sonata-project/SonataMediaBundle#360]
When I try to upload a file bigger than upload_max_filesize a FileNotFoundException is thrown, so users get a 500 error without any information on what is going on and how they can fix it.

I think this should be checked and it should show an alert to the user.

This is the stacktrace

``````
Symfony\Component\HttpFoundation\File\Exception\FileNotFoundException: The file "" does not exist
    at n/a
        in /var/www/nec/vendor/symfony/symfony/src/Symfony/Component/HttpFoundation/File/MimeType/MimeTypeGuesser.php line 115

    at Symfony\Component\HttpFoundation\File\MimeType\MimeTypeGuesser->guess('')
        in /var/www/nec/vendor/symfony/symfony/src/Symfony/Component/HttpFoundation/File/File.php line 87

    at Symfony\Component\HttpFoundation\File\File->getMimeType()
        in /var/www/nec/vendor/symfony/symfony/src/Symfony/Component/HttpFoundation/File/File.php line 64

    at Symfony\Component\HttpFoundation\File\File->guessExtension()
        in /var/www/nec/app/cache/dev/classes.php line 15660

    at Sonata\MediaBundle\Provider\FileProvider->generateReferenceName(object(Media))
        in /var/www/nec/app/cache/dev/classes.php line 15610

    at Sonata\MediaBundle\Provider\FileProvider->doTransform(object(Media))
        in /var/www/nec/app/cache/dev/classes.php line 15752

    at Sonata\MediaBundle\Provider\ImageProvider->doTransform(object(Media))
        in /var/www/nec/app/cache/dev/classes.php line 15220

    at Sonata\MediaBundle\Provider\BaseProvider->transform(object(Media))
        in /var/www/nec/vendor/sonata-project/media-bundle/Sonata/MediaBundle/Form/DataTransformer/ProviderDataTransformer.php line 61

    at Sonata\MediaBundle\Form\DataTransformer\ProviderDataTransformer->reverseTransform(object(Media))
        in /var/www/nec/vendor/symfony/symfony/src/Symfony/Component/Form/Form.php line 994

    at Symfony\Component\Form\Form->normToModel(object(Media))
        in /var/www/nec/vendor/symfony/symfony/src/Symfony/Component/Form/Form.php line 608

    at Symfony\Component\Form\Form->submit(object(Request))
        in /var/www/nec/vendor/symfony/symfony/src/Symfony/Component/Form/Form.php line 638

    at Symfony\Component\Form\Form->bind(object(Request))
        in /var/www/nec/vendor/sonata-project/admin-bundle/Sonata/AdminBundle/Controller/CRUDController.php line 483

    at Sonata\AdminBundle\Controller\CRUDController->createAction()
        in /var/www/nec/vendor/sonata-project/media-bundle/Sonata/MediaBundle/Controller/MediaAdminController.php line 73

    at Sonata\MediaBundle\Controller\MediaAdminController->createAction()
        in  line 

    at call_user_func_array(array(object(MediaAdminController), 'createAction'), array())
        in /var/www/nec/app/bootstrap.php.cache line 2770

    at Symfony\Component\HttpKernel\HttpKernel->handleRaw(object(Request), '1')
        in /var/www/nec/app/bootstrap.php.cache line 2744

    at Symfony\Component\HttpKernel\HttpKernel->handle(object(Request), '1', true)
        in /var/www/nec/app/bootstrap.php.cache line 2874

    at Symfony\Component\HttpKernel\DependencyInjection\ContainerAwareHttpKernel->handle(object(Request), '1', true)
        in /var/www/nec/app/bootstrap.php.cache line 2175

    at Symfony\Component\HttpKernel\Kernel->handle(object(Request))
        in /var/www/nec/web/app_dev.php line 27```
``````

**Comment by Filoz** [comment]
https://github.com/sonata-project/SonataMediaBundle/pull/601

**Comment by jordisala1991** [comment]
Fixed already on #1107
